src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,146 @@
+package duke.ui;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * The UI component for the fanTASKtic.
+ * Handles interactions with the user, such as reading commands and displaying messages.
+ */
+public class Ui {
+
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private Scanner sc;
```

Comment:
You can name your larger scope variables like this more clearly, e.g. `private Scanner scanner`

Tip: You can press Shift+F6 over variables to rename them and IntelliJ will rename all usages of the variable for you.